src/main/java/com/powsybl/openloadflow/ac/AcloadFlowEngine.java

```diff
@@ -118,25 +124,30 @@ public AcLoadFlowResult run() {
         runningContext.lastNrResult = newtonRaphson.run(voltageInitializer);
         double initialSlackBusActivePowerMismatch = runningContext.lastNrResult.getSlackBusActivePowerMismatch();
 
+        runningContext.nrTotalIterations.add(runningContext.lastNrResult.getIterations());
+
         // continue with outer loops only if initial Newton-Raphson succeed
         if (runningContext.lastNrResult.getStatus() == NewtonRaphsonStatus.CONVERGED) {
 
             // re-run all outer loops until Newton-Raphson failed or no more Newton-Raphson iterations are needed
-            int oldIterationCount;
+            int oldCumulatedNrIterations;
```

Comment:
You have kept here oldCumulatedNrIterations.

Reply:
Fixed